ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/DebugTraceBlockByNumber.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright ConsenSys AG.
+ * Copyright contributors to Hyperledger Besu.
```

Comment:
nit: think you used just "Besu" above - is that the latest template?

Reply:
It was auto-generated. I was aware that it changed. I modified all the filed related to this PR with the new template